15_perms_lab/permslab.txt

```diff
@@ -1,15 +1,19 @@
 1. Using the file structure in the current directory, write the commands that perform the following:
 
   - change permission of file src/main.c to r--r--r-- using chmod letter notation
-
+$ chmod u=r,g=r,o=r src/main.c
   - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
-
+$ chmod 564 include/player.h
+Ofir>> shouldn't it be r-xrw-r--?
   - change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
-
+$ chmod 600 Doc/*
   - change permission of "lib" folder and all files and subdirectories inside it to rwxrwxrwx
-
+& chmod 777 lib lib/**
```

Comment:
you can pass the flag ```-R``` to ```chmod``` which means recursive and it will change all permissions for the directory and  it's content (also deeply nested content):```chmod -R 777 lib```